tests/lib/rules/all.js

```diff
@@ -108,10 +108,10 @@ ruleTester.run('_.first', rules['first'], {
   ],
   invalid: [{
       code: '_.first([0, 1, 3])',
-      errors: ['Consider using the native Array.prototype.slice()']
```

Comment:
This `rule.alternative` change shouldn't since it is outside the scope of the primary change. Also, to a lesser extent it no longer (two years later 😅) exists within this file either- and so this commit is the cause of the merge conflict

Reply:
(but -good news- the change you made was later added [here](https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore/blob/6f185d71a86a6fe04e09a34685bcc5e6d84d5085/lib/rules/rules.json#L50-L54))